searchobject/src/main/java/org/jalau/at18/searchobject/controller/ProcessFrameController.java

```diff
@@ -36,7 +39,7 @@ public ResponseEntity<List<MatchInfo>> readDataCriteriaFrame(@RequestParam("file
                                                                  @RequestParam String recipient) throws IOException {
         // save file, it is saving the zip file and getting the path
         Path path = storageService.save(file);
-        System.out.println(path);
+        LOG.info(path.toString());
         // get the route file
         UnzipFile unzip = new UnzipFile(path);
         //FileSource fileSource = new FileSource(path.toFile().getAbsolutePath());
```

Comment:
Perhaps you could remove the commented lines of code if they are not necessary.

Reply:
I didn't do this comments, but i can still remove them if you prefer.

Reply:
Si pudiera por favor :D

Reply:
It is already removed